front/app/modules/commercial/content_builder/admin/utils.ts

```diff
@@ -0,0 +1,11 @@
+import messages from './messages';
+
+const craftComponentNameMap = {
```

Comment:
Mapping the translations to the component names to be able to internationalize them.

Reply:
Mixed feelings about this, we have a script to detect dangling translations (messages that are never used), and these constructions will not be detected in this way.

But I understand the convenience, and how it would make displaying the component name in the toolbox header more difficult. But we could just make that header part of the render function of the actual settings of the component and refer to it explicitly.

Not a biggie, can live with it, just sharing

Reply:
Good point! I did it in this way because this logic is reused in two places but indeed - repeating it is not a huge issue and I would rather keep the intl script working :)